quint/src/cliCommands.ts

```diff
@@ -558,10 +563,22 @@ export async function verifySpec(prev: TypecheckedStage): Promise<CLIProcedure<V
   return verify(config).then(res =>
     res
       .map(_ => ({ ...verifying, status: 'ok', errors: [] } as VerifiedStage))
-      .mapLeft(err => ({
-        msg: err.explanation,
-        stage: { ...verifying, status: 'failure', errors: err.errors },
-      }))
+      .mapLeft(err => {
+        const trace: QuintEx[] | undefined = err.traces ? ofItf(err.traces[0]) : undefined
+        if (trace !== undefined) {
+          // Always print the conterexample, unless the output is being directed to one of the outfiles
```

Comment:
To double-check my understanding:
This is because on verify, we always expect a counterexample to be printed, right?

Reply:
That’s my assumption, yep! This is following the “spec” Igor wrote in the docs for the command: https://github.com/informalsystems/quint/blob/main/doc/quint.md#command-verify